index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Messi Fever!</title>
+
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+ 	</head>
+	 
+	<body>
+		
+		<nav>
+			<a href="index.html"> Home </a> |
+			<a href="experience/resume.html" target="_blank"> Resume and Honors</a> |
+			<a href="http://www.fcbarcelona.com/all-about-leo-messi" target="_blank"> Official Messi site</a> |
+			<a href="http://www.worldsoccershop.com/shop-by-player-messi--lionel.html" target="_blank"> Buy 	Gear </a>
+		</nav>
+
+		<h1>Messi Fever! </h1>
+		<img src="images/lionel_messi_cover.jpeg" text="Lionel Messi" class="main" >
+		<h3>La Pulga "The Flea"</h3>
+		
+
+		<hr/>
+		<h3 class="flea">What's that? The Flea? </h3>
+		
+ 		<p> 
+			Yeah, Messi's nickname is "La Pulga," and that's Spanish for "the flea."
+			To us, it doesn't get much better than that. He's little. He harasses defenders. He never goes away and gets under your skin. It fits. <br/>
+			And anyway, if you can turn a wimpy-sounding nickname like "The Flea" into a descriptor of the 
+			   <a href="https://www.genius.com/Lionel-messi-messis-career-honours-annotated" target="_blank"> 
+					best player in the world 
+			   </a> 
+			, you've clearly done something right.
+		</p>
+
+		<img class="action" src="images/in_action_messi.jpg" text="Lionel Messi" >
```

Comment:
:hammer: We're so close! Your actual filename is `in_action_Messi.jpg`, with the capital M. Your `src` attribute in your HTML needs to be exactly the same. Otherwise, the image won't show.